site/interfaces/index.ts

```diff
@@ -46,3 +46,36 @@ export interface CookiesApiRequest extends NextApiRequest {
         tracking: "on" | "off";
     };
 }
+
+export interface Feedback {
+    question: string;
+    answer: string;
+}
+
+export interface FeedbackApiRequest extends NextApiRequest {
```

Comment:
I'd probably just keep this in the feedback API file since it's only relevant there